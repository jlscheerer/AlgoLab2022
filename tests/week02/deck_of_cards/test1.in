30
200 162
62 87 15 66 25 56 9 14 8 73 55 57 78 75 80 14 56 56 31 73 20 72 51 72 48 78 73 21 58 98 71 55 25 100 18 73 70 48 92 49 48 26 20 41 16 61 56 11 85 64 67 94 6 81 50 53 61 84 71 15 62 78 28 6 13 87 96 59 41 11 78 34 34 31 24 56 96 52 58 34 75 67 48 54 55 14 94 22 82 67 6 97 75 10 7 77 21 90 3 28 41 81 76 36 23 29 81 14 17 11 13 33 62 22 85 32 40 82 10 69 7 20 2 70 19 74 92 28 91 72 45 70 67 65 34 39 15 69 70 100 26 99 16 55 16 87 42 69 3 94 59 25 33 100 80 18 77 94 11 56 71 32 36 69 66 71 31 46 2 38 74 80 18 74 67 61 13 71 70 39 61 3 48 24 15 66 29 94 73 87 52 60 15 100 59 77 1 97 85 41
200 162
29 82 60 22 92 4 1 69 56 15 10 72 51 73 8 71 55 9 87 41 83 14 39 53 80 66 91 9 16 54 60 4 75 22 90 13 13 56 63 94 71 65 91 79 11 62 94 20 73 41 69 39 56 50 68 37 58 8 40 65 8 74 81 51 66 26 61 94 53 1 48 76 3 65 74 91 52 29 32 90 28 9 100 57 97 100 7 89 99 60 22 96 66 30 79 76 75 24 79 7 76 73 62 44 54 48 81 70 68 95 9 91 95 30 74 52 53 31 3 8 3 4 80 7 99 84 35 41 3 73 47 75 1 47 58 32 61 84 17 99 44 52 36 15 61 41 4 58 17 1 24 55 36 94 40 17 1 2 94 57 29 44 30 27 3 71 64 15 56 20 56 51 7 54 8 32 74 23 81 11 24 36 14 68 39 46 21 43 26 65 70 16 61 78 17 84 5 81 8 58
200 74
35 90 32 81 21 83 57 81 96 38 74 54 57 43 5 44 71 73 65 3 8 55 28 61 39 35 43 92 98 17 23 40 24 59 50 78 38 36 26 100 8 44 28 86 65 65 23 45 4 46 56 64 67 53 84 72 16 53 8 88 18 32 66 79 85 35 56 3 60 35 47 76 83 96 56 56 33 86 11 52 88 36 86 7 14 98 36 3 92 8 37 26 55 8 80 68 57 47 55 93 47 12 54 24 53 27 19 80 76 93 7 61 67 52 25 89 22 85 67 54 65 29 47 60 61 89 36 63 79 25 40 11 1 47 98 25 88 100 72 35 29 90 60 91 69 75 28 31 45 2 93 74 47 69 43 84 18 73 80 88 78 81 13 39 93 72 72 50 33 77 73 29 77 83 25 99 72 31 60 32 56 37 14 33 81 97 55 88 63 15 51 7 52 76 100 9 16 13 15 98
200 139
79 73 7 88 51 68 76 21 16 11 22 33 44 36 62 29 74 75 40 55 19 84 71 49 79 23 96 48 47 56 49 19 43 57 58 49 95 11 55 97 45 58 30 18 84 18 12 72 47 36 3 95 17 4 28 61 56 11 10 11 98 56 35 85 20 71 45 67 74 56 21 42 84 85 57 58 70 59 80 8 3 59 43 30 49 82 91 23 99 51 85 15 76 88 29 10 11 21 37 93 83 12 83 35 74 42 53 83 53 41 92 75 37 36 96 71 34 69 31 48 34 7 20 43 69 84 5 82 27 81 49 37 63 24 78 19 88 96 6 2 82 49 54 60 67 15 43 94 79 22 97 83 34 2 12 52 7 36 46 49 7 89 88 43 21 31 15 68 12 36 91 42 34 82 43 92 35 10 89 21 22 72 95 81 90 5 62 1 76 61 84 7 81 7 24 78 6 10 18 74
200 137
15 26 7 27 41 90 60 85 30 6 7 26 19 44 10 79 71 97 100 19 31 68 80 2 79 94 64 94 87 56 17 72 90 85 95 63 73 59 33 45 10 88 80 90 17 90 72 22 3 99 100 43 31 43 30 49 2 80 13 43 59 33 20 8 31 25 27 28 18 24 93 47 38 7 27 61 12 4 7 62 88 64 49 81 18 29 42 33 43 89 33 68 50 58 38 51 53 23 87 75 68 43 79 67 29 25 87 21 65 86 70 17 88 84 98 35 67 27 96 70 53 10 54 71 5 36 73 84 6 46 80 53 70 18 18 48 9 86 10 48 3 7 41 47 52 21 97 1 71 57 83 17 41 75 43 99 82 22 14 42 15 41 79 43 39 41 12 7 93 49 11 88 9 7 91 71 8 16 56 69 41 57 3 75 43 49 41 9 95 85 40 13 94 21 98 12 18 70 53 33
200 166
80 69 72 92 75 50 97 10 95 38 49 88 65 21 31 61 1 53 98 16 21 94 16 52 61 17 18 17 29 99 37 35 95 44 47 38 10 37 19 26 16 1 83 17 31 53 16 83 69 30 8 49 64 85 70 87 38 68 41 5 41 43 51 32 19 75 25 32 15 86 87 68 55 43 63 7 40 61 66 28 13 12 39 35 2 16 22 73 88 51 85 81 15 45 48 15 80 5 16 7 16 88 5 95 99 38 73 27 6 79 52 64 4 33 73 81 98 7 36 55 53 100 62 84 51 84 92 100 38 86 15 47 70 4 53 14 38 90 41 93 89 35 4 68 24 16 100 70 8 68 56 70 91 89 92 35 37 29 95 43 74 28 88 24 94 93 60 24 21 60 6 68 92 85 15 9 32 89 91 54 65 15 61 100 13 96 82 86 94 3 95 93 96 29 52 29 48 52 33 44
200 49
39 60 9 70 23 70 5 97 76 89 66 73 4 58 85 66 2 47 81 1 29 11 73 36 96 15 32 59 52 30 86 29 1 39 42 82 46 30 59 71 41 31 13 74 74 60 77 12 74 71 11 25 82 93 97 100 88 69 36 3 39 28 64 31 93 4 32 18 7 96 78 84 51 76 30 26 39 67 25 16 94 91 66 55 50 26 83 40 82 47 64 81 34 89 5 9 70 93 78 28 35 48 2 23 58 56 42 17 93 9 28 16 47 90 5 14 66 29 29 5 32 51 72 56 46 18 73 21 88 98 18 65 86 33 98 52 53 46 56 17 85 38 53 46 53 19 84 98 94 45 16 72 48 74 75 78 94 93 71 97 65 2 26 66 44 88 69 16 18 98 59 92 39 81 42 78 52 51 97 66 63 31 13 24 11 2 26 36 71 61 40 5 43 15 42 29 38 1 12 94
200 106
62 4 74 19 37 63 69 83 97 32 10 96 72 91 76 5 21 9 31 37 66 69 51 60 68 87 93 84 39 61 22 96 83 6 69 26 69 30 34 5 49 89 62 99 5 87 59 17 85 41 54 100 44 42 70 28 5 26 85 94 89 58 24 4 15 64 54 84 95 73 30 44 23 25 6 70 72 92 26 38 71 74 59 13 86 41 63 71 86 81 6 78 63 85 15 84 24 92 68 20 81 5 7 90 34 35 71 83 93 13 3 89 56 97 8 70 83 23 68 94 58 67 20 61 39 36 83 43 17 26 41 61 1 35 62 62 82 63 61 58 35 71 74 85 99 90 100 97 34 4 46 97 11 82 27 85 39 53 51 91 100 18 1 70 32 76 17 18 97 94 30 41 25 78 79 75 60 17 50 90 2 84 49 72 67 41 42 68 89 94 6 91 72 14 97 83 33 53 72 34
200 15
89 54 58 39 99 52 100 3 61 88 40 99 67 84 23 62 89 36 80 68 35 5 10 21 25 83 43 72 89 43 46 56 27 97 77 8 89 11 6 59 4 60 11 86 71 79 41 67 47 36 25 23 89 7 96 47 50 32 62 49 94 39 59 18 93 4 90 9 70 15 12 74 9 98 77 55 42 92 72 26 15 56 4 19 35 78 96 73 88 23 71 61 62 92 8 52 96 7 8 4 100 10 54 88 4 41 30 82 80 12 55 14 86 50 14 52 46 24 91 17 66 98 51 70 85 17 63 62 42 32 32 31 2 70 27 70 46 44 57 36 20 36 49 52 78 6 74 83 83 1 39 82 31 61 76 7 50 28 23 9 9 52 10 22 49 85 43 11 54 32 57 19 94 15 13 11 93 84 9 30 73 45 42 69 5 2 37 59 3 44 17 86 64 91 88 68 52 68 57 41
200 111
34 21 40 12 8 71 79 44 2 76 49 51 45 76 50 69 64 65 53 7 72 59 96 22 2 62 71 41 16 56 44 54 85 83 34 76 60 82 56 100 14 86 88 57 37 27 90 1 36 36 8 64 82 34 98 89 21 91 91 49 58 33 54 5 56 93 54 68 34 69 40 90 38 96 38 37 50 3 40 91 46 40 49 96 86 4 71 65 22 42 89 80 35 80 68 89 1 62 89 74 19 4 1 8 25 38 48 13 56 48 1 61 91 97 9 1 69 80 92 87 18 7 6 40 88 72 30 54 46 6 86 43 51 89 100 69 42 91 14 40 56 38 15 43 66 4 15 20 89 12 96 79 67 44 74 13 53 1 81 88 82 61 64 35 78 40 62 78 89 13 21 12 95 57 32 65 73 86 94 57 39 29 71 52 21 70 28 68 84 38 92 75 64 50 68 28 89 25 12 65
200 22
79 53 8 26 70 49 15 17 31 28 98 64 22 21 54 82 72 90 4 26 49 30 67 96 67 19 14 78 54 98 65 84 5 75 71 52 17 27 75 69 70 79 12 85 100 5 12 84 12 15 42 91 42 10 43 87 39 15 12 62 18 82 55 48 92 53 27 21 35 78 72 48 79 72 60 77 86 98 16 20 38 80 73 31 56 48 46 66 49 39 75 64 77 96 19 73 85 83 42 46 49 48 81 53 20 77 75 4 62 71 78 75 40 51 56 11 26 10 12 72 35 27 34 37 14 68 42 62 43 72 67 47 36 99 78 33 34 5 45 73 38 12 13 26 16 16 60 22 42 91 75 95 43 26 89 59 75 42 22 36 41 92 74 26 1 9 54 90 93 48 17 4 49 49 83 96 47 7 86 11 70 26 55 35 54 34 43 55 44 51 1 24 10 99 91 7 84 20 5 60
200 96
74 48 2 1 97 48 19 48 49 12 28 29 32 79 7 13 100 49 81 12 16 4 27 11 54 84 79 93 1 30 44 79 95 85 92 82 36 60 10 20 48 50 39 88 7 84 72 23 70 75 94 91 18 16 53 3 99 47 6 69 97 47 27 69 69 5 71 21 40 71 88 40 64 99 73 7 36 16 96 95 27 5 51 86 22 32 37 28 72 8 21 74 96 41 49 93 48 20 86 53 59 100 64 23 84 56 51 40 25 93 18 50 27 94 62 36 59 33 79 74 74 51 76 89 49 72 96 36 83 4 26 87 91 81 43 49 38 59 83 35 61 80 46 67 12 60 17 31 80 11 100 84 58 62 86 64 54 11 1 20 97 35 14 10 89 6 77 83 61 6 5 35 72 95 100 7 94 51 36 20 30 79 11 73 61 57 93 21 3 74 74 67 40 9 69 20 74 56 79 17
200 155
28 7 97 59 15 27 82 38 39 85 9 95 4 69 56 30 61 86 89 23 25 63 41 29 16 16 92 89 8 37 95 35 15 2 25 12 75 5 92 61 17 24 90 13 10 52 81 41 65 54 48 74 41 85 12 39 57 97 84 54 50 92 50 4 100 31 10 97 69 89 87 42 88 84 17 63 18 12 49 61 17 100 38 65 26 14 6 38 55 8 79 49 12 83 65 74 40 91 55 3 4 34 89 50 39 45 18 68 73 80 80 41 51 92 63 37 43 62 42 21 80 27 79 62 25 41 9 94 54 61 26 10 95 72 42 67 85 76 51 78 74 47 45 46 59 6 37 40 9 19 67 54 56 26 39 34 67 65 46 14 82 20 74 74 76 53 26 82 38 53 60 55 90 94 6 11 67 3 30 26 71 62 85 79 57 58 95 76 89 48 98 30 64 14 79 25 100 76 60 97
200 85
9 23 3 10 38 86 35 94 37 60 96 73 50 33 21 67 76 12 1 95 8 73 72 35 85 44 21 32 7 90 58 69 21 31 17 65 55 37 95 44 8 82 33 10 2 23 62 72 42 45 57 6 96 67 27 90 15 2 70 90 25 22 76 84 83 16 94 30 71 61 5 59 2 70 58 22 96 94 25 50 89 15 42 88 61 73 86 76 55 73 47 88 83 89 13 90 42 43 36 32 2 68 28 77 69 4 59 53 79 70 18 52 93 98 38 92 88 51 38 37 26 54 23 89 45 93 92 47 87 12 39 98 47 65 54 29 61 39 53 85 64 70 63 21 64 69 94 48 39 18 40 58 90 15 53 100 61 27 89 97 82 37 34 33 39 55 9 87 93 43 58 80 56 88 100 36 22 49 34 72 31 57 32 84 40 17 3 49 3 18 16 79 40 38 43 52 98 43 71 44
200 55
97 36 58 24 79 24 97 11 44 7 5 100 73 81 19 96 30 87 99 39 32 56 40 70 86 74 7 74 80 24 74 41 29 21 20 25 31 55 38 100 72 2 45 79 93 33 9 55 44 60 41 1 82 76 77 40 28 68 8 88 58 1 91 91 53 11 48 45 97 44 22 28 26 43 75 16 91 28 10 62 51 77 8 66 3 85 71 40 1 77 87 9 16 24 34 15 5 34 71 93 47 18 99 6 7 60 74 23 68 47 25 67 60 49 12 14 7 99 38 63 21 38 12 37 65 54 25 77 50 96 14 83 34 60 27 23 43 98 86 4 48 4 54 82 92 73 54 33 16 6 66 15 10 39 95 10 37 60 75 15 88 9 3 24 5 37 66 72 52 64 73 89 11 8 35 2 22 36 45 3 20 35 95 55 7 14 59 26 7 100 38 49 41 32 31 94 49 57 45 92
200 178
93 46 52 76 1 40 24 41 9 52 29 36 12 85 14 28 67 32 90 17 65 57 98 29 83 3 59 5 10 48 74 57 79 9 79 56 38 96 64 92 68 75 49 90 56 56 13 89 70 28 44 76 29 66 51 22 42 100 62 6 79 20 20 41 96 54 18 38 5 52 18 92 97 79 100 58 31 61 45 66 77 6 48 66 70 40 47 93 83 10 7 14 74 72 7 89 95 54 96 22 88 91 5 61 42 11 25 100 29 33 20 5 34 77 95 2 82 21 60 8 4 95 49 64 62 11 42 94 48 77 67 52 77 70 66 20 50 68 35 27 12 5 49 11 16 89 39 93 73 84 23 23 16 7 22 45 68 16 59 65 93 85 14 28 11 100 74 55 100 31 2 33 49 68 50 80 28 8 15 21 58 78 87 22 50 70 80 31 59 74 81 39 27 29 53 62 17 67 60 7
200 112
19 71 55 10 85 28 62 24 87 97 1 85 52 30 99 41 26 19 76 94 82 33 74 64 47 8 19 78 59 22 55 6 92 70 66 58 23 93 87 9 97 86 53 63 17 82 59 31 89 72 31 7 64 48 37 99 43 35 77 93 32 88 78 68 28 62 25 15 75 5 63 16 84 73 56 36 23 59 80 97 40 60 85 99 93 100 39 86 58 81 20 30 33 75 48 23 21 88 19 21 26 25 83 27 95 58 36 23 97 86 33 49 74 69 41 49 87 16 74 57 19 50 89 80 26 10 27 83 97 13 32 41 81 82 89 49 48 80 42 92 80 78 58 4 51 32 32 45 42 37 80 32 24 100 100 77 42 7 68 81 76 2 34 52 55 56 86 87 94 76 33 57 39 79 47 82 31 59 30 19 54 16 51 35 14 16 45 7 21 97 80 2 55 58 97 35 35 82 61 67
200 142
80 82 74 37 46 17 31 88 39 33 28 28 13 90 78 62 21 31 47 27 31 56 23 39 77 18 59 74 73 9 62 31 31 52 24 39 79 62 45 12 68 70 58 92 39 16 35 71 69 25 28 41 52 64 66 8 4 25 73 66 100 17 43 84 59 19 51 81 50 69 17 62 13 89 46 21 81 32 65 100 59 79 83 58 82 9 36 9 31 64 91 94 17 63 27 7 43 53 34 12 65 82 90 70 69 84 2 6 90 8 51 46 62 17 3 34 21 37 38 34 61 1 66 20 83 100 8 64 66 69 81 37 41 90 13 1 34 66 70 81 92 85 44 8 55 45 47 18 55 92 89 22 39 88 3 34 98 17 99 1 45 40 93 83 78 93 76 94 41 45 45 32 96 49 50 19 99 100 21 64 43 66 44 6 100 83 69 4 61 27 3 78 23 12 25 62 65 4 29 14
200 174
1 53 30 66 34 47 6 35 8 57 4 35 26 96 59 85 46 88 78 61 43 25 65 85 24 28 73 51 77 90 8 55 26 18 33 23 7 21 29 60 27 4 96 93 81 83 2 45 61 27 99 32 43 42 51 5 14 67 50 28 46 98 37 11 47 86 74 57 28 24 16 61 3 38 42 23 52 15 100 54 15 86 20 82 42 100 72 29 15 38 77 77 52 15 84 15 59 34 32 42 35 10 5 78 42 57 64 85 31 53 63 17 69 87 62 89 95 94 32 2 16 3 65 36 69 74 52 14 41 38 35 50 60 100 32 66 12 71 66 53 89 72 72 94 64 62 91 5 6 87 85 29 93 79 72 2 48 36 35 70 49 34 78 29 40 92 78 11 80 89 69 98 93 29 65 10 9 70 28 88 54 99 54 39 78 1 96 85 20 97 73 62 38 49 15 98 2 70 85 46
200 124
38 91 78 76 78 58 44 97 63 25 75 17 12 74 32 27 87 49 54 63 41 95 1 85 88 54 61 78 58 48 7 2 77 36 83 95 41 16 3 28 68 64 93 66 29 59 40 19 78 7 19 63 80 69 52 28 4 55 69 93 16 83 87 63 32 16 44 43 28 23 43 52 27 90 21 78 57 5 59 1 2 35 90 18 65 30 62 13 57 12 99 46 72 100 80 69 95 75 94 4 46 90 2 88 60 92 11 82 67 78 14 36 97 17 83 36 77 100 7 98 51 27 11 25 55 31 95 71 4 4 56 90 82 75 3 18 71 58 29 88 81 97 47 92 17 61 85 53 47 25 75 55 50 68 32 89 64 45 35 52 34 32 6 33 23 27 58 18 94 58 20 86 54 31 52 9 69 81 80 47 8 54 99 87 94 79 51 28 50 93 51 67 28 67 17 32 31 20 33 68
200 134
57 86 92 28 57 1 62 25 90 78 26 12 58 49 17 70 85 99 63 85 61 21 24 41 61 73 34 18 45 77 72 51 85 69 76 65 47 79 77 98 57 50 8 82 38 52 57 60 76 18 9 98 12 67 61 14 100 10 57 79 97 8 30 62 68 68 32 95 17 89 65 19 43 22 63 6 80 75 36 33 99 59 32 64 36 97 33 76 9 43 19 93 83 24 65 21 83 97 92 90 80 95 50 74 67 79 76 10 81 22 29 30 9 26 61 100 84 19 82 81 41 68 97 77 55 91 23 74 52 43 80 57 22 20 43 88 36 75 62 1 38 34 53 68 93 77 14 81 75 11 50 7 15 89 100 47 33 4 67 38 77 69 27 18 72 39 40 71 10 93 75 78 29 74 98 72 72 84 62 48 89 56 56 95 54 39 95 50 26 43 51 26 76 73 10 87 14 36 82 93
200 84
44 11 24 74 87 94 72 22 92 58 28 44 74 9 55 60 14 97 94 82 81 5 23 85 17 84 64 20 80 45 1 41 80 46 58 27 29 68 13 77 86 28 78 16 19 99 77 56 60 8 44 72 13 9 5 22 46 93 15 68 47 89 91 99 63 61 90 21 22 51 10 16 51 42 30 19 1 41 98 37 60 62 49 45 55 31 36 76 31 69 22 62 65 82 67 35 10 54 74 31 52 32 85 52 12 69 4 56 52 36 68 96 58 19 88 32 59 44 50 45 69 93 20 89 82 17 14 72 85 20 65 66 6 21 70 56 81 31 11 61 38 44 10 11 23 71 62 4 53 17 87 36 95 78 71 48 19 78 9 98 45 99 88 40 36 48 41 38 79 34 62 32 54 95 17 28 70 18 95 64 54 87 40 46 15 18 45 75 8 62 80 41 12 44 77 77 22 18 91 99
200 48
11 74 18 96 87 46 18 78 72 2 36 25 80 31 55 12 87 23 98 6 92 35 27 11 8 74 78 95 20 89 79 82 43 4 63 64 11 40 95 17 59 73 3 49 67 43 53 49 55 27 59 42 22 45 74 27 70 33 69 61 63 16 15 65 38 34 100 49 36 17 31 43 89 85 84 7 79 61 24 100 12 19 51 52 10 68 18 81 42 88 54 9 57 72 62 54 52 98 69 95 58 6 85 79 17 88 80 50 36 25 99 39 20 79 49 60 78 6 7 14 29 58 68 95 79 24 31 10 53 74 9 21 88 67 98 52 100 89 90 45 55 7 71 11 2 57 97 23 5 78 54 36 3 13 39 46 21 9 18 66 43 39 9 20 19 14 26 71 49 40 6 85 30 53 91 20 54 22 28 33 43 26 51 15 40 69 15 87 95 34 99 99 4 48 28 100 68 39 12 2
200 85
88 39 43 35 39 9 90 93 7 25 91 98 48 41 76 82 57 76 32 45 58 34 81 42 74 27 84 62 38 37 22 71 75 75 49 10 85 54 51 52 71 19 89 72 21 57 21 30 3 100 46 38 54 54 91 98 9 44 67 92 76 39 74 97 10 1 68 3 24 71 54 7 13 16 26 37 21 50 81 38 50 43 22 44 69 33 48 37 56 65 93 9 66 76 75 41 2 62 9 75 32 22 93 9 52 96 17 36 14 88 51 95 99 59 13 43 25 87 56 73 10 50 43 7 98 89 29 94 79 7 32 61 100 84 91 77 75 98 99 23 23 54 61 57 39 5 93 76 71 90 51 14 29 13 57 52 59 75 6 48 80 30 85 86 93 63 60 39 94 59 51 11 12 74 6 31 74 27 75 60 9 27 13 99 34 27 95 46 48 66 27 19 4 15 47 22 98 51 97 8
200 49
17 60 5 73 17 99 50 40 86 89 42 11 49 75 90 52 37 85 90 71 27 81 87 91 32 86 88 53 74 71 69 15 6 69 60 13 24 84 96 74 42 2 40 16 76 84 42 45 69 94 94 22 74 8 23 88 98 28 37 7 44 89 15 54 50 55 79 96 27 12 53 54 55 63 48 71 22 30 38 61 67 22 61 49 82 97 78 48 80 82 15 67 98 82 22 99 82 51 85 10 72 98 1 19 12 4 39 89 39 11 97 26 50 78 51 90 81 75 94 91 66 73 56 52 59 31 84 48 98 42 4 34 61 98 86 67 36 44 23 54 63 12 94 47 44 19 8 84 70 72 29 26 75 20 83 7 40 27 78 61 94 50 84 95 84 54 65 3 12 25 82 82 86 26 97 32 47 64 88 83 86 66 54 18 42 40 89 37 95 65 29 81 26 62 31 75 53 23 89 56
200 70
40 6 16 69 52 28 28 45 79 54 18 15 2 17 82 53 36 43 100 12 78 96 35 2 21 26 12 67 58 20 77 44 80 28 14 3 78 2 34 21 1 17 95 42 22 61 60 21 89 39 75 99 35 15 50 30 3 54 95 39 8 90 93 87 47 1 17 96 7 11 98 86 87 67 81 8 69 7 48 61 63 32 43 83 33 5 27 3 14 67 35 62 24 80 39 50 28 60 36 53 98 11 61 44 44 6 52 8 33 41 88 31 82 53 69 91 42 58 55 31 78 24 27 41 10 90 40 19 28 71 59 74 51 26 41 21 52 65 59 3 50 71 82 70 94 73 88 62 36 75 69 90 99 8 44 20 9 48 98 73 47 75 29 92 6 68 27 37 78 79 61 72 10 53 90 10 16 1 86 2 49 74 83 99 97 88 34 24 32 13 55 23 27 86 90 100 14 72 49 38
200 153
2 60 6 37 75 3 62 98 36 23 24 93 84 47 70 61 67 68 94 69 14 46 62 61 28 66 79 74 73 24 1 21 81 30 30 15 51 62 46 45 14 97 26 63 50 59 56 75 53 35 57 53 47 33 14 77 32 6 6 87 31 4 70 55 64 97 59 82 11 53 67 1 23 71 97 28 29 70 99 76 54 69 77 59 63 32 6 9 96 55 39 2 38 68 26 63 95 1 9 80 56 61 36 76 34 26 48 34 72 9 23 11 18 29 8 52 25 57 61 72 86 19 34 87 30 16 65 61 52 33 13 39 57 91 53 7 73 86 3 55 17 83 47 98 42 78 7 17 3 21 98 97 39 89 14 62 15 7 56 56 10 83 65 49 74 66 31 27 39 84 66 51 67 4 64 30 84 36 16 78 18 27 38 36 86 51 57 28 34 100 89 14 82 3 74 65 83 39 39 2
200 32
13 18 2 92 54 87 89 15 34 46 35 13 26 86 77 22 20 57 48 72 81 46 6 91 28 68 71 97 16 86 20 1 73 6 38 96 20 85 17 17 71 34 52 27 83 38 96 74 62 37 49 36 19 76 9 47 86 53 98 5 84 9 60 50 13 34 58 6 4 20 33 7 32 63 64 93 91 73 39 38 57 71 22 16 68 35 30 41 44 35 98 24 25 49 15 9 3 35 64 53 17 48 3 54 69 55 45 82 40 37 10 73 88 96 76 39 14 5 62 7 76 98 98 12 4 39 55 59 12 22 71 62 24 7 93 90 76 39 53 13 21 9 33 80 43 73 21 65 10 95 100 14 36 36 5 46 70 6 83 34 89 70 77 15 95 22 77 64 8 28 100 44 72 64 73 11 54 87 92 57 64 12 24 76 12 50 12 77 17 40 13 7 95 25 57 4 5 75 100 91
200 55
70 88 8 47 96 93 6 100 9 2 34 67 50 97 95 58 29 47 32 95 75 15 36 56 76 57 65 48 10 100 71 75 4 63 5 48 85 36 24 7 67 76 90 50 50 73 8 81 17 11 10 72 85 82 64 65 32 21 40 37 14 3 90 22 30 28 70 87 94 67 22 92 100 42 50 89 90 65 100 28 20 63 57 61 52 98 29 53 4 59 7 46 23 7 47 89 61 59 97 17 40 68 48 57 13 75 89 11 10 90 34 34 72 88 48 82 47 27 57 7 24 96 55 23 66 53 48 30 79 50 38 51 58 34 41 84 94 49 54 79 83 53 29 52 88 16 36 42 65 95 55 21 31 28 46 77 46 51 63 58 43 87 83 64 15 41 41 26 85 24 75 17 56 68 74 48 96 38 98 50 44 76 3 47 44 51 13 13 37 97 67 73 34 24 22 2 53 12 47 85
200 78
5 6 92 88 48 6 100 39 57 58 41 35 98 5 56 49 88 97 78 2 10 60 88 9 43 53 2 69 65 71 79 3 88 68 88 100 50 88 6 56 20 29 20 69 85 6 21 62 55 57 48 76 17 46 54 57 66 18 50 96 39 29 20 47 67 44 80 16 2 70 26 23 67 3 56 97 61 77 34 50 86 65 38 18 39 68 62 48 1 72 37 76 77 62 98 70 95 38 27 22 48 26 18 85 95 13 8 1 90 65 89 83 80 63 92 92 40 75 41 92 38 9 29 94 39 8 19 21 51 70 34 2 42 2 44 43 68 15 67 67 24 19 51 86 44 49 58 9 26 50 63 53 41 95 91 32 66 87 79 80 87 83 68 15 79 78 13 30 88 25 90 60 85 31 5 52 16 58 31 33 21 56 57 36 93 31 42 52 98 8 13 15 87 72 5 69 57 99 34 38
