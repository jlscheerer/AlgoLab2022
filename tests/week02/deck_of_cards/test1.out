2 5
91 92
10 10
3 4
37 39
64 68
16 17
10 11
21 22
104 106
12 12
21 24
13 15
24 24
7 8
104 107
52 53
34 36
22 24
57 58
24 25
25 25
69 70
36 36
12 12
143 143
26 27
72 72
2 3
18 18
